sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -522,7 +518,7 @@ static void _send_multicast(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
         }
     }
 
-    _send_multicast_over_iface(iface, pkt, netif);
+    _send_multicast_over_iface(iface, pkt);
```

Comment:
No. This pretends netif to pkt (meaning that pkt is netif afterwards and netif->next the old value of pkt).